cannot find symbol when compile FE with JDK 12

**Describe the bug**
The following error will occur when compile FE with JDK 12:
```
[ERROR] /home/kangkaisen/palo/fe/src/main/java/org/apache/doris/catalog/EsTable.java:[43,22] package javax.xml.soap does not exist
[ERROR] /home/kangkaisen/palo/fe/src/main/java/org/apache/doris/task/LoadPendingTask.java:[41,24] cannot find symbol
  symbol:   class Debug
  location: package sun.security.ssl
```

I found the `sun.security.ssl.Debug` is unnecessary. So we could remove it.

Besides, **I think we should add a [maven-checkstyle-plugin](https://maven.apache.org/plugins/maven-checkstyle-plugin/) to check FE code style.**

--- a/fe/src/main/java/org/apache/doris/task/LoadPendingTask.java
+++ b/fe/src/main/java/org/apache/doris/task/LoadPendingTask.java
@@ -38,8 +38,6 @@ import org.apache.logging.log4j.Logger;
 import java.util.List;
 import java.util.UUID;
 
-import sun.security.ssl.Debug;
-
 public abstract class LoadPendingTask extends MasterTask {
     private static final Logger LOG = LogManager.getLogger(LoadPendingTask.class);
     private static final int RETRY_NUM = 5;

